src/app/utils/highlights.ts

```diff
@@ -112,14 +112,15 @@ export function startCaptureHighlights(): void {
       }
 
       const highlight = {
+        matchId: activeMatch._id,
         videoSrc: cloudinaryURL,
         events: events,
         timestamp: timestamp,
       };
-      const updatedMatch = await postMatchHighlight(highlight, activeMatch._id);
 
-      activeMatch = updatedMatch;
-      console.log('updatedMatch', updatedMatch);
+      const createdHighlight = await postHighlight(highlight);
+
+      console.log('createdHighlight', createdHighlight);
```

Comment:
Is the console.log really necessary, you do nothing with it, except logging it to the console, wouldn`t it be enough to just post it?
```suggestion
      await postHighlight(highlight);
```